docs/develop/worker-performance.mdx

```diff
@@ -336,7 +335,7 @@ const workerOptions = {
   tuner: {
     tunerOptions: resourceBasedTunerOptions,
   },
-}
+};
```

Comment:
nit. No space between brace and next comment?

Reply:
What did you mean? Currently, there are no spaces, you want me to add?

Reply:
It's just a code style nit. Feel free to ignore.